content/openapi/cli-plugin.md

```diff
@@ -184,6 +186,25 @@ export interface PluginOptions {
   </tr>
 </table>
 
+On the first run and when ever plugin options are changed. Delete `/dist` folder and build the application using
+
+```bash
+$ nest build
+```
+
+Or you can configure `package.json` with the following scripts
+
+```json
+"prebuild": "rimraf dist",
```

Comment:
Instead of using `rimraf` direclty we can delegate this to Nest CLI

`nest-cli.json` must have

```json
"compilerOptions": {
  "deleteOutDir": true
}
```

Reply:
A nest app created with Neset CLI using `nest new` generates a package.json with `rimraf` in `prebuild` using that made sense to me. 

The `dist` directory needs to be deleted only when changes to `nest-cli.json` are made and not every time a build is done as is the case with `"deleteOutDir": true` 

Does removing the `dist` directory every time have any impact on the build performance? If not then your suggestion is a lot better to do.

Reply:
> Does removing the `dist` directory every time have any impact on the build performance?

Since the output dir will be removed by Nest CLI only upon `nest build` then both solutions will perform the same way I guess.

I'd say: keep your solution. But it would be cool to also add something like "Or using `"deleteOutDir": true` on [global compiler options](https://docs.nestjs.com/cli/monorepo#global-compiler-options)"

Reply:
sounds good, I've added it